input/simulation_schema.xsd

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
+
+    <xs:element name="configuration">
+        <xs:complexType>
+            <xs:sequence>
+                <xs:element name="fps" type="xs:integer" minOccurs="0" />
+                <xs:element name="video_length" type="xs:integer" minOccurs="0" />
+                <xs:element name="delta_t" type="xs:decimal" minOccurs="0" />
+                <xs:element name="end_time" type="xs:decimal" minOccurs="0" />
+                <xs:choice>
+                    <xs:element name="ds_container" type="xs:string"/>              <!--  ds == direct_sum  and  lc == linked cells  -->
+                    <xs:element name="lc_container">
```

Comment:
```suggestion
                    <xs:element name="linkedcells_container">
```

I would suggest writing out those names, otherwise it is becomes confusing as you cant see the comment which explains the names in the actual  input.xml files

Reply:
Ok sure